<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MsgIDFilterWidget</class>
 <widget class="QWidget" name="MsgIDFilterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="idGrb">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Message ID Filter</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0" colspan="4">
       <widget class="QListView" name="idFilterListView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="filterIDLoadBtn">
        <property name="toolTip">
         <string>Open a Message ID Filter configuration</string>
        </property>
        <property name="toolTipDuration">
         <number>10000</number>
        </property>
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QCheckBox" name="enableIDFilterCheckBox">
        <property name="toolTip">
         <string>Enable the Message ID Filter</string>
        </property>
        <property name="toolTipDuration">
         <number>10000</number>
        </property>
        <property name="text">
         <string>Enabled</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <spacer name="idHorzSpcr">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="filterIDSaveBtn">
        <property name="toolTip">
         <string>Save the current Message ID Filter configuration</string>
        </property>
        <property name="toolTipDuration">
         <number>10000</number>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="addFilterIDPushButton">
        <property name="toolTip">
         <string>Add a new ID to the Message ID Filter</string>
        </property>
        <property name="toolTipDuration">
         <number>10000</number>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="rmvFilterIDPushButton">
        <property name="toolTip">
         <string>Remove the selected ID from the Message ID Filter</string>
        </property>
        <property name="toolTipDuration">
         <number>10000</number>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="3">
       <widget class="QCheckBox" name="invertIDFilterCheckBox">
        <property name="toolTip">
         <string>Invert the Message ID Filter</string>
        </property>
        <property name="toolTipDuration">
         <number>10000</number>
        </property>
        <property name="text">
         <string>Inverted</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
